Dto/UserDto.cs

```diff
@@ -17,4 +19,6 @@ public class UserDto
     public int Points { get; set; }
     public DateTime JoinDate { get; set; }
     public UserStatus OnlineStatus { get; set; }
+    public int[]? AnswersId {get; set;}
```

Comment:
I don't think that we really need to return here int[]? AnswersId and  int[]? QuestionsId